README.md

```diff
@@ -1,14 +1,22 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![Java CI](https://github.com/AY2122S1-CS2103T-W15-3/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/AY2122S1-CS2103T-W15-3/tp/actions/workflows/gradle.yml)
+[![codecov](https://codecov.io/gh/AY2122S1-CS2103T-W15-3/tp/branch/master/graph/badge.svg?token=40MOICZDNE)](https://codecov.io/gh/AY2122S1-CS2103T-W15-3/tp)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+## Introduction
+SoConnect is a desktop application for students :man_technologist::woman_technologist: in [NUS School of Computing](https://www.comp.nus.edu.sg/)
+to **manage the contacts** of their professors and tutors, and **keep track of** their busy schedule of **events**.
+
+## Usage
+- Add, delete, view contact information of professors
+  - Save email addresses, zoom links, office locations and more!
+- Add, delete, view events
+  - Check upcoming events
+  - [Future implementation] Link events to other contacts as participants!
+- View more information at our user guide: [https://ay2122s1-cs2103t-w15-3.github.io/tp](https://ay2122s1-cs2103t-w15-3.github.io/tp)
```

Comment:
Instead of showing the hyperlink as a website, maybe you can give a description such as what you did in line 7? Also, the link doesn't bring us to the user guide though, it brings us to sort of a main page. How about following the original deleted line 13 format?